Packs/FeedTAXII/Integrations/FeedTAXII2/FeedTAXII2.yml

```diff
@@ -72,38 +72,44 @@ configuration:
   name: credentials
   type: 9
   required: false
+- additionalinfo: The API Root to use (e.g. default, public). If left empty, the server default API root will be used. When the server has no default root, the first available API root will be used instead.
+  display: API Root to use
+  name: default_api_root
+  type: 0
+  required: false
 - additionalinfo: Indicators will be fetched from this collection. Run "taxii2-get-collections" command to get a valid value. If left empty, the instance will try to fetch from all the collections in the given discovery service.
   display: Collection Name To Fetch Indicators From
   name: collection_to_fetch
   type: 0
   required: false
 - additionalinfo: Incremental feeds pull only new or modified indicators that have been sent from the integration. As the determination if the indicator is new or modified happens on the 3rd-party vendor's side, and only indicators that are new or modified are sent to Cortex XSOAR, all indicators coming from these feeds are labeled new or modified.
+  defaultvalue: 'false'
   display: Incremental Feed
   name: feedIncremental
-  defaultvalue: 'false'
   type: 8
   required: false
 - additionalinfo: When enabled, fetch-indicators will try to fetch the entire feed for every fetch. When disabled, fetch-indicators will try to fetch just the latest entries (since the last fetch).
-  defaultvalue: 'true'
   display: Full Feed Fetch
   name: fetch_full_feed
   type: 8
   required: false
+  defaultvalue: 'true'
 - additionalinfo: The maximum number of indicators that can be fetched per fetch. If this field is left empty, there will be no limit on the number of indicators fetched.
   display: Max Indicators Per Fetch (disabled for Full Feed Fetch)
   name: limit
   type: 0
   required: false
-- additionalinfo: The time interval for the first fetch (retroactive). <number> <time unit> of type minute/hour/day/year. For example, 1 minute, 12 hour
-  defaultvalue: 1 year
-  display: First Fetch Time
+- display: First Fetch Time
   name: initial_interval
+  defaultvalue: 1 year
   type: 0
   required: false
+  additionalinfo: 'The time interval for the first fetch (retroactive). <number> <time unit> of type minute/hour/day/year. For example, 1 minute, 12 hour'
```

Comment:
Recommend:
'The time interval for the first retroactive fetch, formatted as <number> <time unit of type minute/hour/day/year>. For example: 1 minute, 12 hours.'